pandas/io/pytables.py

```diff
@@ -3972,8 +3972,7 @@ def write(self, obj, axes=None, append=False, complib=None,
 
         # create the axes
         self.create_axes(axes=axes, obj=obj, validate=append,
-                         min_itemsize=min_itemsize,
-                         **kwargs)
+                         min_itemsize=min_itemsize, **kwargs)
```

Comment:
Minor nit but would be good not to include formatting tweaks like this in your PR to keep diffs small (unless directed to by flake8 or a developer)

Reply:
I will revert this, but this actually wasn't an intentional reformatting change. This was one of the functions that I originally removed `**kwargs` from but had to put it back to make tests pass. Didn't realize I changed the formatting in the process.